docs/ml-features.md

```diff
@@ -26,7 +26,9 @@ This section covers algorithms for working with features, roughly divided into t
 
 `HashingTF` is a `Transformer` which takes sets of terms and converts those sets into 
 fixed-length feature vectors.  In text processing, a "set of terms" might be a bag of words.
-The algorithm combines Term Frequency (TF) counts with the 
+A binary toggle parameter controls term frequency. When set to true all nonzero frequencies are
+set to 1. This is especially useful for discrete probabilistic models that model binary counts
+rather than integer. The algorithm combines Term Frequency (TF) counts with the
 [hashing trick](http://en.wikipedia.org/wiki/Feature_hashing) for dimensionality reduction.
```

Comment:
It might better to switch the order of these sentences, so you describe the algorithm first, then the optional binary parameter.
```
The algorithm combines...
A binary toggle parameter...
```